lab-david/frontend/src/lib/redux-thunk.js

```diff
@@ -0,0 +1,2 @@
+export default store => next => action => 
```

Comment:
you need a redux-reporter.js here instead with a try catch that sets state

Reply:
I removed the redux writer because we weren't using it.